src/Widgets/TestimonialSliderWidget/testimonialSliderWidget.scss

```
@@ -0,0 +1,45 @@
+@import "src/assets/stylesheets/variables";
+@import "src/assets/stylesheets/bootstrap/functions";
+@import "src/assets/stylesheets/bootstrap/variables";
+@import "src/assets/stylesheets/bootstrap/mixins/transition";
+
+.quote-headline {
```

Please change the css namespace to the widget name and its function. So in this case:

```suggestion
.testimonial-slider-widget {
  &--quote-headline {
```